src/HealthChecks.UI/Core/HostedService/HealthCheckReportCollector.cs

```diff
@@ -115,15 +123,15 @@ private async Task<UIHealthReport> GetHealthReportAsync(HealthCheckConfiguration
 
                         using var requestMessage = new HttpRequestMessage(HttpMethod.Get, absoluteUri);
                         requestMessage.Headers.Authorization = new BasicAuthenticationHeaderValue(userInfoArr[0], userInfoArr[1]);
-                        response = await _httpClient.SendAsync(requestMessage, HttpCompletionOption.ResponseHeadersRead);
+                        response = await _httpClient.SendAsync(requestMessage, HttpCompletionOption.ResponseHeadersRead).ConfigureAwait(false);

```

Comment:
Please remove all added `.ConfigureAwait(false)` to simplify review. I did not add them intentionally across UI projects.

Reply:
It's been fixed :)